commonmark-ext-yaml-front-matter/src/main/java/org/commonmark/ext/yaml/YAMLFrontMatterBlock.java

```diff
@@ -0,0 +1,6 @@
+package org.commonmark.ext.yaml;
```

Comment:
Can you change the packages to `org.commonmark.ext.front.matter` instead? (Seems more future proof, apparently there's other front matter formats, e.g. JSON.)